usagov/usagov-2021
# USAGOV-1473 language toggle tab order

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1473

## Description

This change moves the language toggle markup into the navbar element and removes the old languageswitcher Drupal block.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



- Shrink your browser window to view the mobile layout
- Visit any page with language toggles
- Tab through the header items
  - [x] Confirm that the focus-order goes from the USAgov logo to the language toggle and then to the mobile menu button.
- Adjust your window size to view the desktop layout
  - [x] Confirm that the focus-order goes from the USAgov logo to the language toggle and then to the  phone number.
- Visit a page without language toggles (like the elected officials results pages)
  - [x] Confirm that the navbar looks decent without the language toggle
- Try both English and Spanish pages

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
